modules/tinymce/src/themes/silver/main/ts/ui/statusbar/ResizeHandle.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Behaviour, Dragging, Focusing, Keying, SimpleSpec, Tabstopping } from '@ephox/alloy';
+import { Optional } from '@ephox/katamari';
+import { SugarPosition } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
+import { get as getIcon } from '../icons/Icons';
+import { resize, ResizeTypes } from '../sizing/Resize';
+
+const getResizeType = (editor: Editor): ResizeTypes => {
+  // If autoresize is enabled, disable resize
+  const fallback = !editor.hasPlugin('autoresize');
+  const resize = editor.getParam('resize', fallback);
+  if (resize === false) {
+    return ResizeTypes.None;
+  } else if (resize === 'both') {
+    return ResizeTypes.Both;
+  } else {
+    return ResizeTypes.Vertical;
+  }
+};
+
+const keyboardHandler = (editor: Editor, resizeType: ResizeTypes, x: number, y: number): Optional<boolean> => {
+  const scale = 3;
+  const delta = SugarPosition(x * scale, y * scale);
+  resize(editor, delta, resizeType);
+  return Optional.some(true);
+};
+
+export const renderResizeHandler = (editor: Editor, providersBackstage: UiFactoryBackstageProviders): Optional<SimpleSpec> => {
+  const resizeType = getResizeType(editor);
+  if (resizeType === ResizeTypes.None) {
+    return Optional.none();
+  }
+
+  return Optional.some({
+    dom: {
+      tag: 'div',
+      classes: [ 'tox-statusbar__resize-handle' ],
+      attributes: {
+        'title': providersBackstage.translate('Resize'), // TODO: tooltips AP-213
+        'aria-hidden': 'true'
```

Comment:
This will need removing, as you can't have something that's focusable while also being hidden as users using a screen reader will have it navigate to it and not say anything.